### ISSUE TITLE
Accessibility: Lack of Visible Keyboard Focus

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1042]
# Accessibility: Lack of Visible Keyboard Focus

## Issue
All of the "Start Earning", "View Bounties", and "Sign In" buttons on the landing page are missing visible focus indicatoin when interacting with a keyboard.

## Expected Behavior
- WCAG 2.2 AA [(2.4.7 Focus Visible)](https://www.w3.org/TR/WCAG22/#focus-visible) requires all actionable elements that are navigable via keyboard to have a visible focus indicator.
- Enable a focus outline which is both perceivable and usable when a user Tabs to the interactive control with a keyboard

## Acceptance Criteria
- [x] Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.
- [x] Enable a visible focus outline (must also meet 3:1 contrast ratio as per [(1.4.11 Non-Text Contrast)](https://www.w3.org/TR/WCAG22/#non-text-contrast))
- [x] Test using keyboard tab navigation on desktop and mobile (enable FKA - Full Keyboard Access on MacOS/iOS)
- [x] Ensure that each interactive element receives a visible focus outline when tabbing through the page
- [x] Ensure that focus follows a logical order (left-to-right and top-to-bottom) 
- [x] Preferably also follows [Success Criterion 2.4.13 Focus Appearance](https://www.w3.org/TR/WCAG22/#focus-appearance) 

_Linked to: [#1040 - Tracking Issue - Accessibility Review: Bounties Landing Page](https://github.com/stakwork/sphinx-tribes-frontend/issues/1040)_

**Comment by jason-me** [comment]
PR in pending review status for completion of this issue: https://github.com/stakwork/sphinx-tribes-frontend/pull/1054

**Comment by jason-me** [comment]
Closed by: https://github.com/stakwork/sphinx-tribes-frontend/pull/1054
